Documentation/installation/k8s-install-broadcom-vmware-esxi-nsx.rst

```diff
@@ -0,0 +1,99 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _k8s_install_broadcom_vmware_esxi_nsx:
+
+******************************************
+Installation on Broadcom VMware ESXI / NSX
+******************************************
+
+Cilium can be installed on VMware ESXI with or without NSX by using official image
+
+Deploying Cilium on Broadcom VMware vSphere ESXi with or without NSX(-T)
+========================================================================
+
+Cilium can be deployed on VMware vSphere ESXi, with or without NSX(-T). However, there are known issues when using VXLAN as the encapsulation mode.
+
+Known Issue: Pod Communication Failure Across Hosts
+===================================================
+
+When deploying Cilium with VXLAN encapsulation, inter-host pod communication may fail, except for ICMP (ping), which still functions.
+
+To check if you are affected by this issue, you can run the following command inside a Cilium pod:
+
+.. code-block:: shell-session
+
+    kubectl exec -n kube-system <cilium-pod> -- cilium--health status --verbose
+
+Alternatively, you can use ``k8s-cilium.sh`` to inspect the cluster state refer to the :ref:`troubleshooting_k8s`
+
+Additionally, you might encounter the following error related to HTTP probes: ``context deadline exceeded (Client.Timeout exceeded while awaiting headers)``
```

Comment:
The troubleshooting guide has this suggestion already, but also the underlying problem behind a `cilium-health status` error being reported might not be the offloads issue. It could be, but it could be caused by something else. I wonder whether we need to repeat these troubleshooting steps in an install guide? Or maybe it's enough to just rely on the existing troubleshooting docs.